apps/library/linker/link.common.ld

```diff
@@ -5,11 +5,11 @@ MEMORY
 {
     instrram    : ORIGIN = 0x00000000, LENGTH = 0x1000
     dataram     : ORIGIN = 0x00001000, LENGTH = 0x0E00
-    stack       : ORIGIN = 0x00001E00, LENGTH = 0x0100
+    stack       : ORIGIN = 0x00001E00, LENGTH = 0x0200
```

Comment:
Why are there 2 different files for linker?
apps/library/linker/link.common.ld
apps/link.common.ld

Reply:
apps/library/linker/link.common.ld unnecessary, taken care of.